api/android/api/src/main/jni/nnstreamer-native.h

```diff
@@ -179,23 +179,29 @@ nns_destroy_pipe_info (pipeline_info_s * pipe_info, JNIEnv * env);
 extern void
 nns_set_priv_data (pipeline_info_s * pipe_info, gpointer data, nns_priv_destroy destroy_func);
 
+/**
+ * @brief Get element data of given name.
+ */
+extern element_data_s *
```

Comment:
Q: Is this ```extern``` keyword required? In general, the ```extern``` keyword is implicitly assumed when a function is declared in the header file.